### PR TITLE
Add the setting page in the dev branch.

### PR DESCRIPTION
**Pull request by LucasAub**
Add the settings page on the dev branch to have the latest version of the application on this branch.

### CODE REVIEW

File: adopte_a_candidate/lib/pages/log_in.dart
Comment (MaximeAlgosup) [inline adopte_a_candidate/lib/pages/log_in.dart:1]: We should not have 2 differents widgets for the page to differentiate the language

File: adopte_a_candidate/lib/pages/profile_page.dart
Comment (MaximeAlgosup) [inline adopte_a_candidate/lib/pages/profile_page.dart:1]: The profile page is too long and too messi, it should be divided in several widgets to improve the readability

File: adopte_a_candidate/lib/pages/settings.dart
Comment (MaximeAlgosup) [inline adopte_a_candidate/lib/pages/settings.dart:1]: The setting page is too long and too messi, it should be divided in several widgets to improve the readability

File: adopte_a_candidate/lib/pages/sign_up.dart
Comment (MaximeAlgosup) [inline adopte_a_candidate/lib/pages/sign_up.dart:1]: We should not have 2 differents widgets for the page to differentiate the language

**Review by MaximeAlgosup — CHANGES_REQUESTED** [review]
Many things must be changed before taking this version as a work base. So I refuse the request

**Review by Clementine951 — APPROVED** [review]
Some changes are needed on the SettingsPage branch, but it doesn't bring any crash or issues.

**Review by MaximeAlgosup — APPROVED** [review]
Not everything add been changed, but since it's supposed to be change in the next pull request I accept this one

**Pull request merged**